docker-compose.ha.yml

```diff
@@ -392,29 +387,64 @@ services:
       - "/mnt/NASVolume/msa_sms_logs:/opt/sms/logs"
       - "/mnt/NASVolume/msa_bulkfiles:/opt/sms/spool/parser"
```

Comment:
**/mnt/NASVolume/msa_monitbulkfiles** instead of **/mnt/NASVolume/msa_bulkfiles** for lines 388 and 389